alpha-build-machinery/make/targets/openshift/deps.mk

```diff
@@ -2,7 +2,7 @@ self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
 scripts_dir :=$(self_dir)/../../../scripts
 
 # We need to force localle so different envs sort files the same way for recursive traversals
-DIFF :=LC_COLLATE=C diff
+deps_diff :=LC_COLLATE=C diff --no-dereference -N
```

Comment:
the `--no-dereference` makes the difference

Reply:
links in repos are always fun